staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protoc-gen-gogo/main.go

```diff
@@ -37,6 +37,9 @@ func main() {
 	for _, file := range request.FileToGenerate {
 		if strings.Contains(file, "/") {
 			param := "paths=source_relative"
+			if request.Parameter != nil && *request.Parameter != "" {
+				param = *request.Parameter + "," + param
```

Comment:
It is possible for the param to get very long if you keep concatenating. Should there be a check here to ensure that scenario doesn't happen.

Reply:
We break the loop after concatenating the params only once.  How can we check and handle if this concatenation leads to param being too long? I am not sure what exceptions will occur in this case.

Reply:
> It is possible for the param to get very long if you keep concatenating. Should there be a check here to ensure that scenario doesn't happen.

We break the loop after concatenating the params only once. How can we check and handle if this concatenation leads to param being too long? I am not sure what exceptions will occur in this case.

Reply:
Good point on breaking the loop after one iteration. That should prevent the parameter concatenation from getting out of hand. I think that is reasonable.

Reply:
I think merging params could get weird... would it be better to output an error in this case and tell the invoker to add `paths=source_relative` to their params?

Reply:
> I think merging params could get weird... would it be better to output an error in this case and tell the invoker to add `paths=source_relative` to their params?

It seems to be the most reasonable.  :)